ros/src/computing/perception/detection/vision_detector/packages/vision_darknet_detect/CMakeLists.txt

```
@@ -157,6 +157,53 @@ IF (CUDA_FOUND)
             LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
             RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
             )
+ELSEIF(EXISTS "ENV{HOME}/darknet")
```

I think the $ is missing
```suggestion
ELSEIF(EXISTS "$ENV{HOME}/darknet")
```